tools/govulncheck-parser/main.go

```
@@ -0,0 +1,106 @@
+		return ResultNotReachable, nil
+	}
+
+	return ResultNotFound, nil
+}
+
+const expectedArgs = 2
+
+func main() {
+	if len(os.Args) != expectedArgs {
+		fmt.Fprintf(os.Stderr, "Usage: %s <CVE-ID>\n", os.Args[0])
+		os.Exit(2)
```

golangci-lint (2.12.2)

[error] Magic number: 2, in <argument> detected

(mnd)